src/test/java/tests/elements/factory/IFindElementsTests.java

```diff
@@ -65,7 +65,7 @@ default void testShouldBePossibleToWorkWithElementsFoundBySpecificLocator(By loc
     default Object[] getSupportedLocators() {
         List<By> locators = new ArrayList<>();
         locators.add(CalculatorWindow.getDottedXPathLocator());
-        locators.add(CalculatorWindow.getTagNameLocator());
+        locators.add(CalculatorWindow.getTagXpathLocator());
```

Comment:
It seems that currently there are 3 locator strategies supported. Ideally we need to add By.className to cover them all
https://appium.io/docs/en/drivers/windows/
![image](https://user-images.githubusercontent.com/9014926/145072584-afa6aafb-9c1f-45dd-a4a8-918bc3fe9ad9.png)